relayer/relays/beacon/header/header.go

```diff
@@ -71,7 +71,7 @@ func (h *Header) Sync(ctx context.Context, eg *errgroup.Group) error {
 
 	log.Info("starting to sync finalized headers")
 
-	ticker := time.NewTicker(time.Second * 10)
+	ticker := time.NewTicker(time.Minute * 1)
```

Comment:
The beacon relayer can check for new updates every minute, since we only sync sync committee updates. Can even make this even 10 minutes or so.